<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradesAdding</class>
 <widget class="QWidget" name="GradesAdding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(64, 115, 175);</string>
  </property>
  <widget class="QWidget" name="widget_AddingGrade" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(219, 226, 239);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);</string>
   </property>
   <widget class="QPushButton" name="pushButton_Add">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(18, 46, 79);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 75 20pt &quot;Gentium Book Basic&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Add </string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Subjects">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_YourGrade">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MaxGrade">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Note">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>371</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NessesaryField_1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>55</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
font: 14pt &quot;Gentium Book Basic&quot;;
color:red;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NessesaryField_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>55</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
font: 14pt &quot;Gentium Book Basic&quot;;
color:red;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NessesaryField_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>135</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
font: 14pt &quot;Gentium Book Basic&quot;;
color:red;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearSubjects">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:white;
background-color:rgb(18, 46, 79);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_YourGrade">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Your grade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_AddingGrade">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Adding grade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MaxGrade">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Max grade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Subject">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Subject:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Date">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Note">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Note:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SubjectName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
